src/instana/instrumentation/redis.py

```diff
@@ -43,6 +43,8 @@ def execute_command_with_instana(
         args: Tuple[object, ...],
         kwargs: Dict[str, Any],
     ) -> object:
+        if check_if_ignored("redis", args[0]):
+            return
```

Comment:
Let's prevent the load and import of all these classes. Move this if block out of the try-except block and all the imports (between lines 8 and 10) to below the `import redis`.